src/utils/Utilities.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export function toogleFullscreen(ref: React.RefObject<HTMLDivElement>) {
+  if (!ref.current) {
+    return;
+  }
+  if (document.fullscreenElement) {
+    document.exitFullscreen();
+  } else {
+    ref.current.requestFullscreen();
```

Comment:
We should `catch` in case of denied permission and maybe show an error toast with `notifications.show`

Reply:
Ahh... is there a permission to be challenged .... have not done frontend stuff in a long time. Will check.

Reply:
Yup, frontend is always a fatigue.. feel you!